user-guide/Advanced_Modules/DOM/DOM_objects/DOM_FieldDescriptors/DOM_ElementFieldDescriptor.md

```diff
@@ -0,0 +1,42 @@
+---
+uid: DOM_ElementFieldDescriptor
+---
+
+# ElementFieldDescriptor
+
+- **FieldValue type**: string of `ID` of an `Element` ([DMA ID]/[ELEMENT ID])
```

Comment:
```suggestion
- **FieldValue type**: string (Example: "686/1245")
```
The format is described in the description. Maybe can add an example value to make it already clear here as well.